features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/operationComponents/AddParameter.jsx

```diff
@@ -80,6 +84,18 @@ function AddParameter(props) {
     React.useEffect(() => {
         setLabelWidth(inputLabel.current.offsetWidth);
     }, []);
+
+    React.useEffect(() => {
+        const isParameterExistValue = operation.parameters && operation.parameters.map(operations =>
+            (!!((operations.in === newParameter.in && operations.name === newParameter.name))));
```

Comment:
@kavishkafernando , Why do we need double not (**!**) here?

Reply:
fixed